README.md

```diff
@@ -1 +1,41 @@
-# classify-tutorials-surveys
\ No newline at end of file
+# classify-tutorials-surveys
+
+This project builds a Random Forest classifier which determines whether research papers are suitable by attempting to determine whether they are tutorial or survey articles. The module finds training data by mining google scholar, mag, and other sources for papers and associated metadata. Then, it trains the classifier on selected features. Using the trained classifier, we can once again query research sources for suitable articles, determining whether an arbitrary article is suitable based on its output when put in to our classifier.
+
+## Functional Design
+* Queries for articles related to keyword, returns papers dict with metadata for each article
+* Saves paper into accessable format (likely .csv) for classifier to access and train from
+```python
+fetch_papers(keyword_list):
```

Comment:
I think it would be helpful to let users input the file location / directory / name where they want articles to be stored.